### ISSUE TITLE
Extract birth/death place for wikibase-person

### ISSUE THREAD

**Issue by chsamlee** [infolab-csail/WikipediaBase#245]
I found a comment in `tests/test_resolvers.py` that dates back the v1.1.0:
"# TODO: should DEATH-PLACE be a calculated attribute of wikibase-person?"

If this feature were to be implemented, the implementation will focus on first paragraph (especially on the parentheses) similar to BIRTH-DATE. In terms of difficulty BIRTH-PLACE should be easier than BIRTH-DATE, but I'm not sure how much code can be reused.

Is the usefulness of this method (compared to using infobox) high enough to justify actually implementing this?

**Comment by sfelshin** [comment]
I don't think this feature is worth implementing at this time, so I've changed the priority to "low".  We might want this feature someday, but first we would want to investigate a bit to see if there seem to be enough people who have an extractable birth/death place in the first sentence but not in the infobox (e.g., people who don't have an infobox).
